lib/ansible/modules/files/stat.py

```diff
@@ -294,7 +300,8 @@
             sample: ../foobar/21102015-1445431274-908472971
             version_added: 2.4
         md5:
-            description: md5 hash of the path
+            description: md5 hash of the path, this will be removed in Ansible 2.9 in
+                favour of the checksum return value
```

Comment:
doh, the Queen's English creeps in (s/favour/favor)... `,` should probably be `;` as well